dashboard/app/models/levels/level.rb

```diff
@@ -602,6 +602,19 @@ def age_13_required?
     false
   end
 
+  # Add a starter asset to the level and save it in properties.
+  # Starter assets are stored as an object, where the key is the
+  # friendly filename and the value is the UUID filename stored in S3:
+  # {
+  #   # friendly_name => uuid_name
+  #   "welcome.png" => "123-abc-456.png"
+  # }
+  def add_starter_asset(friendly_name, uuid_name)
```

Comment:
it seems like this code (and the `starter_assets` in the `serialized_attrs` list) would better belong in dashboard/app/models/levels/applab.rb, since assets are specific to applab. They might better fit here if we thought we might generalize to support other level types, but AFAIK we aren't going to need this for gamelab or other levels. apologies for not catching this in the previous PR.

Reply:
that's okay! i think that's a great idea to have `starter_assets` live on Applab instead of the Level model. i'll do this in a follow-up PR